recipes/combined-pvalues/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   md5: 2ed642df0222a36dfce13e98c35f1b9f
 
 build:
-  noarch: pip
+  noarch: python
```

Comment:
Oh gosh ...

Reply:
Yeah, that killed the nightly upload. That likely happened when I merged in the bulk "fix the linting" changes. I guess when you change hundreds of packages at once you'll muck a few up :P